workflow-templates/im-deploy-windows-service.yml

```diff
@@ -11,16 +11,24 @@
 #    workflow but it will be streamlined (instructions below) since the Deploy Multiple Items workflow handles many of these checks.
 #
 # Projects to use this Template with:
-#    - On-Prem Service (Core Template)
+#    - Windows Service (Core Template)
 #
 # TODO: Prerequisites:
 #     - Make sure the Service account secrets have been added to the environment
 #     - Make sure the host windows server has been prepped to accept incoming WinRM Requests
 #     - Ensure each of the repo-level and env-level secrets used in this workflow have been populated by an admin in your repository.
 #     - Set up a deployment board if it does not already exist: https://github.com/im-practices/git-er-done/blob/main/actions/deployment-board.md
 #       Generally one board should be set up per windows service in the repository.
-
-name: Deploy On Prem Windows Service # TODO: If there are multiple on prem services in the repo, append the service name to this workflow's file name and update this name
+#     - The target server must be domain joined. If this is not the case already, create an ITHD ticket to have it joined to
+#       the ExtendHealth domain for service account authentication
+#     - Create an ITHD ticket to have an ExtendHealth Active directory service account created for each deployment environment.
+#       That service account should then be added to an AD Group Policy Object (GPO); that GPO will then be applied to target
+#       servers/deployment environments
+#     - Do not use the instructions in the windows server action to prep the target server for WinRM connections,
```

Comment:
Similar to another comment, can this point take the place of point number 2 that seems to be saying something similar?